components/ResourceYaml.vue

```diff
@@ -256,13 +256,29 @@ export default {
       @onReady="onReady"
       @onChanges="onChanges"
     />
-    <Footer v-if="!isView" :mode="mode" :errors="errors" @save="save" @done="done">
```

Comment:
Doesn't this make view always show a cancel button?

Reply:
Yes, I found it odd we display the buttons everywhere but on view, but I am going to roll this back. It should be changed everywhere at once.